aladdin-container.sh

```diff
@@ -75,13 +75,28 @@ function exec_command_or_plugin() {
 }
 
 function _replace_aws_secret() {
+    if ! test -z "$(get_or_set_cache "aws_secret_${CLUSTER_NAME}")"; then
```

Comment:
Also, this would be a place where it might be night to have a separate get_cache.  No need for the reader to wonder what that function is doing here and whether it's changing anything.